test/framework/src/main/java/org/elasticsearch/index/mapper/FieldMapperTestCase.java

```diff
@@ -203,8 +179,8 @@ public void testMergeConflicts() {
         }
         {
             FieldMapper mapper = (FieldMapper) newBuilder().build(context);
-            FieldMapper toMerge = new FieldMapper("bogus", new BogusFieldType(), new BogusFieldType(),
-                SETTINGS, FieldMapper.MultiFields.empty(), FieldMapper.CopyTo.empty()) {
+            FieldMapper toMerge = new FieldMapper("bogus", new MockFieldMapper.FakeFieldType(),
```

Comment:
I think this could just be `FIeldMapper toMerge = new MockFieldMapper("bogus")` ?

Reply:
It could, thanks!